packages/component-library/src/CivicSandboxMap/CivicSandboxMap.js

```diff
@@ -11,6 +11,7 @@ import DeckGL, {
   ScreenGridLayer,
 } from 'deck.gl';
 import { css } from 'emotion';
+import * as d3 from 'd3';
```

Comment:
I'm trying to move away from importing * as d3 to importing individual modules from d3 so we get the benefits of that modularization. We're doing it some other places, so maybe it makes sense to do this as a separate issue